src/analysis/analysis.py

```diff
@@ -138,6 +124,35 @@ def main(data_filepath, out_filepath):
         column='Model:N'
     ).save(os.path.join(out_filepath,'model_performance.png'))
 
+def train_random_forest(X_train, y_train, preprocessor, scoring_metrics, cross_val_results):
+    pipe_rf = make_pipeline(preprocessor, RandomForestClassifier(class_weight='balanced', random_state=123))
+    cross_val_results['random forest'] = pd.DataFrame(cross_validate(
+        pipe_rf, X_train, y_train, n_jobs=-1, scoring=scoring_metrics)).agg(['mean']).round(3).T
+
+def train_svc(X_train, y_train, preprocessor, scoring_metrics, cross_val_results):
+    pipe_svc = make_pipeline(preprocessor, SVC(class_weight='balanced', random_state=123))
+    cross_val_results['svc'] = pd.DataFrame(cross_validate(
+        pipe_svc, X_train, y_train, n_jobs=-1, scoring=scoring_metrics)).agg(['mean']).round(3).T
+    cross_val_results['svc']
```

Comment:
I don't think this does anything? OR does it display the results to the command line? If it does this I don't think we want to display results to the command line by default. We could set it as a flag in the function call, something like:
```suggestion
    # set the default value to True or False, whichever makes sense
    train_svc( . . . , display_results= False):
        . . .
        if display_results is True:
             cross_val_results['svc']
        . . .        
```

Reply:
No actually this is training the model. It was not intended to display anything except for the UserWarning.

Reply:
Ok cool, so then what does line 136 of the original analysis.py script do?

Reply:
It was for notebook display. I got rid of them in the new commit